djoser/views.py

```diff
@@ -202,7 +202,7 @@ def resend_activation(self, request, *args, **kwargs):
         user = serializer.get_user(is_active=False)
 
         if not settings.SEND_ACTIVATION_EMAIL or not user:
-            return Response(status=status.HTTP_400_BAD_REQUEST)
+            return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Maybe change this to a "positive check"? I.e.:

```python
if settings.SEND_ACTIVATION_EMAIL and user:
    context = {"user": user}
    to = [get_user_email(user)]
    settings.EMAIL.activation(self.request, context).send(to)
```

That way this method has only one point of return, this seems to be more inline with how the other methods are coded.

Reply:
Good idea. Updated.